### PR TITLE
Change release-1.5 draft notes doc uris to final

### PR DESCRIPTION
**Pull request by spiffxp**

### CODE REVIEW

**Comment by saad-ali** [comment]
/lgtm

Devin?

**Comment by idvoretskyi** [comment]
LGTM

File: release-1.5/release-notes-draft.md [hunk 1]
```diff
@@ -20,41 +20,41 @@ Features for this release were tracked via the use of the [kubernetes/features](
...
+  - [beta] Deployments that cannot make progress in rolling out the newest version will now indicate via the API they are blocked ([docs](http://kubernetes.io/docs/user-guide/deployments/#failed-deployment)) ([kubernetes/features#122](https://github.com/kubernetes/features/issues/122))
+  - [beta] StatefulSets allow workloads that require persistent identity or per-instance storage to be created and managed on Kubernetes. ([docs](http://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/)) ([kubernetes/features#137](https://github.com/kubernetes/features/issues/137))
+  - [beta] In order to preserve safety guarantees the cluster no longer force deletes pods on un-responsive nodes and users are now warned if they try to force delete pods via the CLI. ([dos](http://kubernetes.io/docs/tasks/manage-stateful-set/scale-stateful-set/)) ([kubernetes/features#119](https://github.com/kubernetes/features/issues/119))
```
Comment (enisoc) [inline release-1.5/release-notes-draft.md]: While you're at it, could you fix dos->docs on this line?

**Comment by devin-donnelly** [comment]
LGTM.

File: release-1.5/release-notes-draft.md (hunk 1 above)
Comment (spiffxp) [inline release-1.5/release-notes-draft.md]: done

**Comment by saad-ali** [comment]
/lgtm merging

Note that these links will not be live until we merge the 1.5 docs branch later today

**Pull request merged**